Laba2SCCS/src/BusinessLayer/Interfaces/IService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace BusinessLayer.Interfaces
+{
+    public interface IService<TModel>
+        where TModel : class
+    {
+        Task<int> Create(TModel item);
+
+        Task Update(TModel item);
+
+        Task Delete(TModel item);
+
+        Task<IEnumerable<TModel>> GetAll();
+
+        Task<TModel> GetById(int id);
```

Comment:
All methods that return Task should have a name with Async suffix (example GetByIdAsync).
Rename it in ALL places (includes private methods in services if they return Task).